config/config.go

```diff
@@ -31,6 +31,8 @@ const (
 	defaultTreeFileBufferSize       = 4096
 	defaultEstimatedLeavesPerSecond = 78000
 	defaultMaxRoundMembers          = 1 << 32
+	defaultSubmitFlushInterval      = 100 * time.Millisecond
```

Comment:
the optimal would likely be different, it is hard to tell though, some experiments with observability are useful for that